src/scope.ml

```diff
@@ -468,9 +469,9 @@ let scope_rule : sig_state -> p_rule -> sym * pp_hint * rule loc = fun ss r ->
   let names = Array.of_list (List.map fst map) in
   let vars = Bindlib.new_mvar te_mkfree names in
   let rhs =
-    let prv = sym.sym_expo = Private in
     let map = Array.map2 (fun n v -> (n,v)) names vars in
-    let t = scope (M_RHS(Array.to_list map, prv)) ss Env.empty p_rhs in
+    let t = scope (M_RHS(Array.to_list map, is_private sym))
+              ss Env.empty p_rhs in
```

Comment:
This is not proper indentation.

Reply:
Fixed in d7b99a6f1252f9d51d90833cc70c9ff9774a01b8.